docs/source/basics/custom_components.rst

```diff
@@ -0,0 +1,171 @@
+.. _custom_components_label:
+
+=============================
+Custom Components and Recipes
+=============================
+
+torchtune lets you launch fine-tuning jobs directly from the command-line using both library and custom datasets,
+models, recipes, and configs. This is done with the ``tune run`` command (see :ref:`cli_label`), which can also be
+used from your project folder.
+
+Setting up your torchtune project
+---------------------------------
+First, ensure that you have torchtune installed, see :ref:`install_label`. This will also install the ``tune`` command
+to your environment.
```

Comment:
```suggestion
First, ensure that you have torchtune installed, see :ref:`install_label`. This will also enable the ``tune`` command
in your environment.
```